tools/cellranger.wdl

```diff
@@ -2,6 +2,7 @@
 ##
 ## This WDL file wrap the [10x Genomics Cell Ranger](https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/what-is-cell-ranger) tool.
 ## Cell Ranger is a tool for handling scRNA-Seq data.
+#@ except: LineWidth
```

Comment:
I assume this means that a lint directive anywhere in the preamble applies to the file? I've been putting them on the first line, but I think it makes sense that it is implemented as anywhere in the preamble.

Reply:
My understanding (without having reviewed the code very closely) is that a lint directive anywhere in the preamble will work. I think my preferred style would be to place directly above the version statement (i.e. either the first line if there are no preamble comments, or separated from the preamble comments by a blank).

```
## Cell Ranger blahblahblah

#@ except: LineWidth
version 1.1

task foo {}
```
But this currently sets off the PreambleWhitespace rule.